test/e2e/tests/test_ca_activation.py

```diff
@@ -44,6 +44,16 @@ def create_secret(k8s_secret):
     )
     yield secret
 
+@pytest.fixture(scope="module")
+def create_certificate_chain_secret(k8s_secret):
+    secret = k8s_secret(
+        "default",
+        random_suffix_name("certificate-chain-secret", 50),
+        "certificateChain",
+        "test"
+    )
+    yield secret
```

Comment:
There is a way to clean up secrets after each test run https://github.com/aws-controllers-k8s/rds-controller/blob/main/test/e2e/fixtures.py#L37-L59